Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs

```diff
@@ -768,25 +690,17 @@ private async Task<bool> HasLineup(ListingsProviderInfo info, CancellationToken
 
             _logger.LogInformation("Headends on account ");
 
-            var options = new HttpRequestOptions()
-            {
-                Url = ApiUrl + "/lineups",
-                UserAgent = UserAgent,
-                CancellationToken = cancellationToken,
-                LogErrorResponseBody = true
-            };
-
-            options.RequestHeaders["token"] = token;
+            using var options = new HttpRequestMessage(HttpMethod.Get, ApiUrl + "/lineups");
+            options.Headers.TryAddWithoutValidation("token", token);
 
             try
             {
-                using (var httpResponse = await Get(options, false, null).ConfigureAwait(false))
-                using (var response = httpResponse.Content)
-                {
-                    var root = await _jsonSerializer.DeserializeFromStreamAsync<ScheduleDirect.Lineups>(response).ConfigureAwait(false);
+                using var httpResponse = await Send(options, false, null, cancellationToken, HttpCompletionOption.ResponseHeadersRead).ConfigureAwait(false);
```

Comment:
I don't think `HttpCompletionOption.ResponseHeadersRead` should be used here?

Reply:
I went through and updated all requests that had `BufferContent = false` without looking carefully at them, so I'm sure there are some that aren't required

Reply:
Please fix it ;)